src/components/expanded-state/UniqueTokenExpandedState.js

```diff
@@ -321,7 +330,18 @@ const UniqueTokenExpandedState = ({ asset, external, lowResUrl }) => {
             </SheetActionButtonRow>
           )}
 
-          {!isPoap && (
+          {asset.network === AssetTypes.polygon && (
+            <L2Explainer
```

Comment:
nit: this is the only file that renames `L2Disclaimer` to `L2Explainer` - can we keep it consistent and just use `L2Disclaimer`